test/HexFloat.cpp

```diff
@@ -1054,8 +1054,8 @@ FloatParseCase<T> GoodFloatParseCase(std::string literal, bool negate_value,
 INSTANTIATE_TEST_CASE_P(FloatParse, ParseNormalFloatTest,
                         ::testing::ValuesIn(std::vector<FloatParseCase<float>>{
                             // Failing cases due to trivially incorrect syntax.
-                            BadFloatParseCase<float>("abc", false),
-                            BadFloatParseCase<float>("abc", true),
+                            BadFloatParseCase<float>("abc", false, 0.0f),
```

Comment:
We no longer need to explicitly specify the parameter type for the `*FloatParseCase` methods -- the compiler should auto-resolve it based on the argument types.